### PR TITLE
Add presses and change name of task to "fulcrum_published_dois"

### PR DESCRIPTION
**Pull request by sethaj**
HELIO-3118
* ran this on preview, works are expected
* once this is merged some crons need to be updated, don't close the ticket.

### CODE REVIEW

File: lib/tasks/handles/fulcrum_published_dois.rake [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      children << Press.where(subdomain: subdomain).first&.children&.pluck(:subdomain)
+    end
+
+    subdomains = subdomains + children
+    subdomains = subdomains.flatten.compact.uniq
+
+    hits = ActiveFedora::SolrService.query("{!terms f=press_sim}#{subdomains.map(&:downcase).join(',')}", rows: 100_000)
+
+    hits.each do |hit|
+      next unless hit["suppressed_bsi"] == false && hit["visibility_ssi"] == "open"
```
Comment (conorom) [inline lib/tasks/handles/fulcrum_published_dois.rake]: Not really a nitpick but I'm always leery of this `suppressed_bsi` thing. I'm pretty sure action is only ever taken on this in Hyrax/stack code when it's true (Thus removed from results). I have sort of a vague memory of seeing it missing from Solr docs in the past, but I may have made this up. Is that possible?
Either way, I'd like if this read `hit["suppressed_bsi"].blank?` but would welcome correction or discussion.

Comment (gkostin1966) [reply lib/tasks/handles/fulcrum_published_dois.rake]: Sounds good to me.  Should added it to rubocop.

Comment (gkostin1966) [reply lib/tasks/handles/fulcrum_published_dois.rake]: Also. I would prefer /open/i.match?(hit["visibility_ssi"]), love those case insensitive regular expressions.

Comment (conorom) [reply lib/tasks/handles/fulcrum_published_dois.rake]: thinkin-face

**Pull request merged**